CAFMaker/CMakeLists.txt

```diff
@@ -21,6 +21,11 @@ art_make_library( LIBRARY_NAME ${PACKAGE}
                             RawData
                             RecoBase
 			    Simulation
+			    ChannelMap
```

Comment:
Just checking, did it fail to build without these in this section? I know the ones below for the module are definitely needed, but can never remember when things need to be pulled into the library ahead of time.

Reply:
it was not needed so I deleted it now